iris_grib/_load_convert.py

```diff
@@ -43,6 +43,7 @@
     ReferenceTarget
 from iris.util import _is_circular
 
+from . import _confirm_iris_mercator_support
```

Comment:
Sorry I this is bugged.
I think it should really be `from ..  import ` ??
Always a bit confused by relative imports ! :confused:

Reply:
I think it works though, but now it can't find `grib_phenom_translation`.  Weird...